### ISSUE TITLE
how can we train on a day and night dataset

### ISSUE THREAD

**Issue by Aryan-Solanki** [csjliang/LPTN#10]
how to devide images in test,train and A,B..plzz explain in details

**Comment by Aryan-Solanki** [comment]
why have you devided train into A,B and how do i have to devide my training set if i m using my own set of images

**Comment by Aryan-Solanki** [comment]
and how can i output only one img not all 3

**Comment by csjliang** [comment]
Hi. For both train and test sets, A denotes the input images (in day2night task, day images) and B denotes the target images (in day2night task, night images). You just need to divide the images into two folders. To process a single image, you can either write code to read one image or just put only one image into the test/A folder.

**Comment by Aryan-Solanki** [comment]
![IMG-20210530-WA0006.jpg](https://user-images.githubusercontent.com/37494973/120108185-b2d24a00-c181-11eb-832f-7aab753fb46e.jpg)

When i pass one img in the model it outputs a img in 3 section...where do i have to make changes so i get only the processed output not all 3img in one img

**Comment by csjliang** [comment]
In the file ```lptn_test_model.py```, change Line 107 to ```imwrite(result_img, save_img_path)```. Thanks.

**Comment by Aryan-Solanki** [comment]
do i have to do any specific code changes or i can just devide Image into A(day imgs) and B(night) folder , change file locations in train_FiveK.yml and run Train.py

**Comment by Aryan-Solanki** [comment]
one more doubt in training the day and night images don't have to be of same location??? random day and night images will work??

Do i have to make every img of 480x720 or 720x480 size??

**Comment by csjliang** [comment]
> do i have to do any specific code changes or i can just devide Image into A(day imgs) and B(night) folder , change file locations in train_FiveK.yml and run Train.py

Just prepare dada and change dataroot in the yml file then you can get a basic model that can translate day images to night. Of course, it will be better if you tune the parameters according to your data. Thanks.

**Comment by csjliang** [comment]
> one more doubt in training the day and night images don't have to be of same location??? random day and night images will work??
> 
> Do i have to make every img of 480x720 or 720x480 size??

Aligned day to night training pairs (captured at the same location) are hard to collect. If you have them, the performance will be far better by using the MSE loss directly. In this project, the translation is mainly supervised by GAN. Therefore, only unpaired data (random day and night images as you said) is required. The images (both training and testing) can be any size in our code, yet please make sure to crop a relatively large patch for each image (change the ```gt_size``` para in the yml file).

**Comment by Aryan-Solanki** [comment]
![image](https://user-images.githubusercontent.com/37494973/120140538-65dd8a80-c1f8-11eb-8ede-95811938d380.png)
why is it showing 5 days ..
i m running on gtx1660ti and my taskmanager is also showing low gpu usage 
![image](https://user-images.githubusercontent.com/37494973/120140657-9b827380-c1f8-11eb-88dc-c45569969bde.png)

![image](https://user-images.githubusercontent.com/37494973/120140765-da182e00-c1f8-11eb-9251-0c0b36061d1a.png)

m i missing something??

**Comment by Aryan-Solanki** [comment]
i m not running the 
CUDA_VISIBLE_DEVICES=0 python codes/train.py -opt options/train/LPTN/train_FiveK.yml
 as it gives error
![image](https://user-images.githubusercontent.com/37494973/120141013-4c890e00-c1f9-11eb-8da0-d4bbd8f4ff6b.png)
![image](https://user-images.githubusercontent.com/37494973/120141220-b30e2c00-c1f9-11eb-891a-9bc410d3737d.png)
 i m direclty running train.py and set the default opt parameter

**Comment by csjliang** [comment]
Please try to directly run the code following our instructions, i.e., run ```PYTHONPATH="./:${PYTHONPATH}" CUDA_VISIBLE_DEVICES=0 python codes/train.py -opt options/train/LPTN/train_FiveK.yml```.
